### ISSUE TITLE
Event search bar is not populated when findings are opened

### ISSUE THREAD

**Issue by brikelly** [getappmap/vscode-appland#818]
Loom is [here](https://www.loom.com/share/76130bd26a4b420d8a9202b74ae131bd?sid=264ba892-c747-4144-a296-1bfe510f1bf5).

This bug makes it impossible to iterate over each event in a finding in the Trace View.

**Comment by dustinbyrne** [comment]
Look at "related events". It should contain the same information previously displayed in the search bar, albeit in a more clear and flexible manner.

**Comment by brikelly** [comment]
@dustinbyrne Yeah I'd seen that, and with the "eyeball" click it's not too bad, but we still lost some control that was there before.
